### PR TITLE
Darkmode 1.2

### PR DESCRIPTION
**Pull request by annatraussnig**
Generalise darkmode throughout the whole site.

### CODE REVIEW

File: components/Crowdfunding/Bar.js [hunk 1]
```diff
@@ -45,20 +45,18 @@ const styles = {
...
     left: 5,
     right: 5,
-    backgroundColor: colors.secondary,
-    color: '#fff',
     fontFamiliy: fontFamilies.sansSerifMedium,
     fontSize: 14,
     lineHeight: '19px',
-    padding: '12px 17px'
+    padding: '12px 17px',
+    color: '#fff'
```
Comment (ovbm) [inline components/Crowdfunding/Bar.js:59]: static color?

File: components/Events/Detail.js [hunk 2]
```diff
@@ -91,10 +94,14 @@ const Event = withT(
       tweet: title,
       shareOverlayTitle: t('events/share/title')
     }
+    const borderRule = colorScheme.set('borderTopColor', 'divider')
 
     return (
       <div {...styles.container}>
-        <div {...styles.block}>
+        <div
+          {...styles.block}
```
Comment (ovbm) [inline components/Events/Detail.js:102]:
```suggestion
          {...styles.block}  {...borderRule}
```

File: components/Events/Detail.js [hunk 3]
```diff
@@ -91,10 +94,14 @@ const Event = withT(
...
+          {...colorScheme.set('borderTopColor', 'divider')}
```
Comment (ovbm) [inline components/Events/Detail.js:103]:
```suggestion
```

**Review by ovbm — COMMENTED** [review]
first part of review. phew...

File: components/Payment/Form.js [hunk 4]
```diff
@@ -112,7 +114,8 @@ const styles = {
   }),
   paymentMethodText: css({
     lineHeight: '40px',
-    verticalAlign: 'middle'
+    verticalAlign: 'middle',
+    color: colors.light.text
```
Comment (ovbm) [inline components/Payment/Form.js:118]:
```suggestion
```

File: components/Payment/Form.js [hunk 5]
```diff
@@ -89,7 +90,8 @@ const styles = {
     marginBottom: 20,
     marginTop: 10,
     '& svg': {
-      marginRight: 5
+      marginRight: 5,
+      marginBottom: -2
```
Comment (ovbm) [inline components/Payment/Form.js:94]: negative margins can be tricky.

File: lib/routes.js [hunk 6]
```diff
@@ -42,7 +42,6 @@ routes
   .add('questionnaireCrowd', `/umfrage/:slug(${routes.questionnaireCrowdSlug})`)
   .add('questionnaire', '/umfrage/:slug')
   .add('events', '/veranstaltungen')
-  .add('event', '/veranstaltung/:slug', 'events')
```
Comment (ovbm) [inline lib/routes.js:45]: why remove this?

File: pages/manifest.js [hunk 7]
```diff
@@ -18,7 +17,11 @@ import {
   mediaQueries
```
Comment (ovbm) [inline pages/manifest.js]:
```suggestion
  mediaQueries,
  useColorContext
```

File: pages/manifest.js [hunk 8]
```diff
@@ -18,7 +17,11 @@ import {
...
 } from '@project-r/styleguide'
 
-import { PUBLIC_BASE_URL, CDN_FRONTEND_BASE_URL } from '../lib/constants'
+import {
+  PUBLIC_BASE_URL,
+  CDN_FRONTEND_BASE_URL,
+  useColorContext
```
Comment (ovbm) [inline pages/manifest.js]:
```suggestion
```

Comment (ovbm) [reply pages/manifest.js]: Screenshot 2021-01-27 at 17 51 52

**Review by ovbm — CHANGES_REQUESTED** [review]
nearly done

File: components/Payment/Form.js (hunk 4 above)
Comment (annatraussnig) [reply components/Payment/Form.js:118]: this is needed for the `rechnung` text on the third payment method tile (https://republik.ch/angebote?package=ABO_GIVE). Background is white and left to its own devices `rechnung`gets rendered in light grey in dark mode.

Comment (ovbm) [reply components/Payment/Form.js:118]: got it!

**Review by ovbm — APPROVED** [review]

**Review by ovbm — APPROVED** [review]

File: components/Vote/201907/BudgetChartItem.js [hunk 9]
```diff
@@ -85,101 +85,89 @@ const styles = {
...
+        {...colorScheme.set('borderBottomColor', 'default')}
+      >
         <div
-          {...styles.wrapper}
+          {...styles.toggle}
+          onClick={hasMore ? toggleCollapsed : undefined}
           style={{
-            background,
-            borderBottom: collapsed ? '1px solid #fff' : undefined
+            color: highlight ? colorScheme.getCSSColor('text') : color,
```
Comment (tpreusse) [inline components/Vote/201907/BudgetChartItem.js:127]: `getCSSColor` isn't allowed to be used in a style object, but you should be able to easily apply it to the div:
```js
<div {...colorScheme.set('color', highlight ? 'text' : color)} />
```

File: components/Article/Page.js [hunk 10]
```diff
@@ -325,7 +325,6 @@ const ArticlePage = ({
 
   return (
     <Frame
-      colorSchemeKey={colorSchemeKey}
```
[the remark this answers is not in the document]
Comment (annatraussnig) [reply components/Article/Page.js:328]: you are obviously correct here.

File: components/Frame/index.js [hunk 11]
```diff
@@ -119,9 +117,10 @@ const Frame = ({
     })
   }, [hasSecondaryNav])
   return (
-    <ColorContextProvider root colorSchemeKey='auto'>
-      <ColorHtmlBodyColors colorSchemeKey={colorSchemeKey} />
-      <ColorSchemeSync />
+    <ColorContextProvider colorSchemeKey={pageColorSchemeKey}>
+      {pageColorSchemeKey && (
+        <ColorHtmlBodyColors colorSchemeKey={pageColorSchemeKey} />
```
[the remark this answers is not in the document]
Comment (annatraussnig) [reply components/Frame/index.js]: seems to be working alright

**Review by tpreusse — APPROVED** [review]
🚢 🇮🇹  Looking good 👏 🎉 

Thank you Anna

**Pull request merged**